integration-tests/utils/integration-utils.js

```diff
@@ -277,11 +277,16 @@ const openError = async page => {
   return !!errorDetails.length
 }
 
-const maybeSaveScreenshot = async (page, testName) => {
+const getScreenshotDir = () => {
   const dir = screenshotDirPath ?
     screenshotDirPath :
     process.env.SCREENSHOT_DIR || process.env.LOG_DIR || resolve(__dirname, '../test-results/screenshots')
+  mkdirSync(dir, { recursive: true })
+  return dir
+}
 
+const maybeSaveScreenshot = async (page, testName) => {
+  const dir = getScreenshotDir()
   try {
     mkdirSync(dir, { recursive: true })
```

Comment:
Shouldn't need this because `getScreenshotDir` makes the directory.

Reply:
Yeah forgot to remove it. Thanks for catching that.